src/e2e_test/perf_tool/experimentpb/experiment.proto

```diff
@@ -62,7 +62,16 @@ message WorkloadSpec {
 message DeployStep {}
 
 // HealthCheck specifies how to run a single check on a workload.
-message HealthCheck {}
+message HealthCheck {
+  oneof check_type {
+    K8SPodsReadyCheck k8s = 1 [ (gogoproto.customname) = "K8S" ];
+  }
+}
+
+// K8sPodsReadyCheck checks that all of the pods in the given namespace have a status of RUNNING.
+message K8SPodsReadyCheck {
```

Comment:
Nit: Capitalization `K8sPodsReadyCheck` and just `K8s` above.

Reply:
If I do `K8sPodsReadyCheck` gogoproto turns it into `K8SPodsReadyCheck` in the .pb.go